src/SA/netuse/entry.c

```diff
@@ -19,11 +20,22 @@
 		}
 
 
-DWORD Net_use_add(LPWSTR pswzDeviceName, LPWSTR pswzShareName, LPWSTR pswzPassword, LPWSTR pswzUsername, BOOL bPersist)
+DWORD Net_use_add(LPWSTR pswzDeviceName, LPWSTR pswzShareName, LPWSTR pswzPassword, LPWSTR pswzUsername, BOOL bPersist, BOOL bPrivacy, BOOL bIpc)
 {
 	DWORD			dwResult	= ERROR_SUCCESS;
 	LPNETRESOURCEW	lpnrLocal	= NULL;
-	DWORD			dwFlags		= CONNECT_TEMPORARY;
+	DWORD			dwFlags		= NULL; //CONNECT_TEMPORARY;

```

Comment:
Changing this to 0 instead of NULL will eliminate the compilation error.
```suggestion
	DWORD			dwFlags		= 0; //CONNECT_TEMPORARY;

```